src/Service/Font/FontLoader.re

```diff
@@ -86,6 +83,7 @@ let loadAndValidateEditorFont =
       };
       let maybeBoldFont =
         family
+        // XXX: Should I make the bold one step upper than the current weight ?
```

Comment:
That sounds good - similar to the mention above, we could have a helper function to get the 'bolder' font weight